### ISSUE TITLE
Fuel records not acting correctly

### ISSUE THREAD

**Issue by T00ManySecrets** [hargata/lubelog#607]
**Checklist**
Please make sure you have performed the following steps before opening a new bug ticket, change `[ ]` to `[x]` to mark it as done

- [x] I have read and tried the steps outlined in the [Troubleshooting Guide](https://docs.lubelogger.com/Troubleshooting)
- [x] I have searched through existing issues.

**Description**
Fuel records behave strange. Im writing in 20 liters or 20.0 in this case and it acts as if i wrote 20000

**Platform**
- [x] Docker Image

**Browser Console Errors(F12)**
No errors

**App/Container Console Error**
No errors

**Screenshots(optional)**
![Screenshot 2024-09-05 065000](https://github.com/user-attachments/assets/0712ca22-fcbf-4f7c-a0e6-aae0c563b33d)
![Screenshot 2024-09-05 065030](https://github.com/user-attachments/assets/69754732-37e3-4557-ac58-a5ad18f834da)

**Comment by hargata** [comment]
Not a big, you’re using a locale which has “,” as the decimal separator, the fuel tab uses 3 decimal places so 20,0 is rendered as 20,000

in US locale this will be 20.0 and 20.000

**Comment by kalipso-cyber** [comment]
> Not a bug, you’re using a locale which has “,” as the decimal separator, the fuel tab uses 3 decimal places so 20,0 is rendered as 20,000
> 
> this is user configurable in the settings tab.
> 
> in US locale this will be 20.0 and 20.000

Hi,

first of all thanks for the app - I just recently set it up and I'm quite liking it!

**TL;DR is that I want to display the number of litres as 34,92 and not 34,920 or 3492,000.**

A bit more content: My LubeLogger has the "Use Imperial Calculation" as well as the "Use UK MPG Calculation" settings both turned off. The "Use three decimals for fuel cost" option is also deactivated. The problem is that my fuel consumption is still shown with three post-comma digits.

So for instance, in an entry in the "Fuel" tab of a vehicle, where I added 34,92 litres of fuel to a vehicle, under "Consumption" it shows as _34,920_. If I enter _34.92_ instead of _34,92_, the _34,920_ changes to _3492,000_. But I don't want to show this with three digits, especially since the third digit will always be 0 anyways as no gas station around me counts three digits after the comma.

Which setting do I need to change to correctly display the number of litres with only two digits after the comma?

Here's a visual representation of my settings:

<img width="1294" alt="settings" src="https://github.com/user-attachments/assets/d19546eb-9066-4be7-87fb-c15886bbb57a">

And this is what the field in question currently looks like:

<img width="1286" alt="fuel consumption" src="https://github.com/user-attachments/assets/fbb32003-c6dc-4cf5-a2f6-70f88036d35f">
